.travis.yml

```diff
@@ -1,13 +1,10 @@
 language: ruby
 
 rvm:
-  - 2.0
   - 2.1
-  - 2.2
+  - 2.2.3
+  - 2.3.0
 gemfile:
-  - gemfiles/Gemfile.ar-3.2
-  - gemfiles/Gemfile.ar-4.0
-  - gemfiles/Gemfile.ar-4.1
   - gemfiles/Gemfile.ar-4.2
```

Comment:
We can use `ActiveRecord::Type.lookup` in case for Rails 5 and keep own map for Rails 4.2. How does it sound for you?